### ISSUE TITLE
dapr stop does not terminate application service (mac)

### ISSUE THREAD

**Issue by macon** [dapr/cli#734]
## Expected Behavior

`dapr stop -a myApp` should stop the application service `myApp`

## Actual Behavior

The above command prints output suggesting it worked but the service is still running.
The dapr-managed service no longer shows up in `dapr list`.

## Steps to Reproduce the Problem

`dapr run --app-id myApp --app-port 5001 -d ./components --  dotnet run -p ./src/myApp.myApp.csproj`

Now check your service listening on `5001` e.g.

```
> lsof -PiTCP -sTCP:LISTEN | grep 5001
Mon  7 Jun 11:13:32 2021 dotnet    74156 someuser  127u  IPv4 0xe1acb3e8e6c0d4f1      0t0  TCP *:5001 (LISTEN)
```

`dapr stop -a myApp`

Check services again and I see the same process (74156) listening on `5001`.

## Info
```
> dapr --version
CLI version: 1.2.0
Runtime version: 1.1.2
```

## Release Note

RELEASE NOTE:

**Comment by wcs1only** [comment]
So, what's happening here: 
`dotnet run` spawns `dotnet exec` child process. When you run `dapr stop -a myApp` the `dotnet run` process is SIGKILL which orphans the `dotnet exec` child process. I have confirmed this should be fixed by #695

**Comment by mukundansundar** [comment]
related to #695
